src/KM_Game.pas

```diff
@@ -75,6 +75,7 @@   TKMGame = class
     DoGameHold: Boolean; //Request to run GameHold after UpdateState has finished
     DoGameHoldState: TGameResultMsg; //The type of GameHold we want to occur due to DoGameHold
     SkipReplayEndCheck: Boolean;
+    IsDefaultGameSpeedInReplayMP: Boolean;  // While watching MP replay we can set to default game speed as it was in MP game
```

Comment:
I don't think this is necessary. If you've sped up the replay why should it suddenly return to the default speed at peacetime? The viewer can do that themselves.